### PR TITLE
Space Ninja smoke bomb refill

### PR DESCRIPTION
**Pull request by Metroidlover**
:cl: MetroidLover
add: Added the ability to gain smoke bomb charges by attacking the initiated ninja suit with a beaker containing smoke powder
/:cl:

This change was added to equalize this with the Ninja Boost ability as the Boost had a way to refill charges, but the Smoke Bomb, an arguably weaker ability, did not have a way to refill it's uses.

### CODE REVIEW

File: code/modules/ninja/suit/suit_attackby.dm [hunk 1]
```diff
@@ -11,6 +11,15 @@
 			to_chat(U, "<span class='notice'>There are now [a_boost] adrenaline boosts remaining.</span>")
 			return
 
+
+	if(istype(I, /obj/item/reagent_containers/glass))//If it's a glass beaker.
```
Comment (YPOQ) [inline code/modules/ninja/suit/suit_attackby.dm]: Duplicate of the previous if

**Comment by MrDoomBringer** [comment]
good first pr! though the changelog should have your name, rather than the pr name. like this:
image
also nice name

**Comment by Metroidlover** [comment]
Thanks, I've never done this before, so I appreciate any feedback.

File: code/modules/ninja/suit/suit_attackby.dm (hunk 1 above)
Comment (Metroidlover) [reply code/modules/ninja/suit/suit_attackby.dm]: Ah, I see what you mean. If you would like, I can change it around to look less spaghetti-ish.

**Comment by Iamgoofball** [comment]
>buffing ninja 

The madman

**Comment by Metroidlover** [comment]
@Iamgoofball 😜 

To be honest, the adrenal refill is much stronger than this, and it uses a base chem dispenser reagent.

**Review by Cyberboss — APPROVED** [review]

**Comment by Cyberboss** [comment]
Keep the changelog mostly for user facing changes please

**Pull request merged**